src/main.c

```diff
@@ -47,7 +46,8 @@ difficulty pickDifficulty() {
     }
 }
 
-char * pickWord(difficulty current_difficulty) {
+char const* pickWord(Difficulty current_difficulty) {
+    char const* randomWord;
```

Comment:
`randomWord` should ideally be `random_word` (it is a variable, not a function), but aside from that; this variable was previously declared in the header despite being used only within one function, and without a need to preserve the value of the variable across multiple function calls. 

Variables should be declared in the minimal required scope/lifetime.